xinference/core/worker.py

```diff
@@ -594,10 +597,14 @@ async def launch_builtin_model(
         launch_args.pop("kwargs")
         launch_args.update(kwargs)
 
-        event_model_uid, _, __ = parse_replica_model_uid(model_uid)
+        try:
+            origin_uid, _, _ = parse_replica_model_uid(model_uid)
+        except Exception as e:
+            logger.exception(e)
```

Comment:
This seems useless.

Reply:
Just in case parse wrong replica_model_uid causing exception,  @log_async does not log exception